### PR TITLE
remove bootstrap3-typeahead

### PR DESCRIPTION
**Pull request by nstadelmaier-dev**
Removes bootstrap3-typeahead lib.

The library lists 2 different licenses therefore it has to be removed form the project.

### CODE REVIEW

File: org.eclipse.winery.repository/src/main/webapp/jsp/servicetemplates/boundarydefinitions/boundarydefinitions.jsp [hunk 1]
```diff
@@ -499,7 +491,6 @@
 						contentType: "application/json"
 					}).done(function (result) {
 						vShowSuccess("Sucessfully created interface");
-						updateInterfaces();
```
Comment (koppor) [inline org.eclipse.winery.repository/src/main/webapp/jsp/servicetemplates/boundarydefinitions/boundarydefinitions.jsp]: Why is this removed? If I change the interface from `x` to `y`, the operations of `y` have to be shown, not the one of `x` kept.

Comment (nstadelmaier-dev) [inline org.eclipse.winery.repository/src/main/webapp/jsp/servicetemplates/boundarydefinitions/boundarydefinitions.jsp]: This method used to update the list of interfaces in case a new interface was created, so that the list contains the new interface as well.
Since the list or typeahead feature for the the provided interfaces and operations was removed, the update for the list is not required anymore. Therefore it was removed.

**Comment by koppor** [comment]
Can you please change the target branch to [release-preparation](https://github.com/eclipse/winery/tree/release-preparation)?
